.github/workflows/release.yml

```
@@ -18,4 +18,51 @@ jobs:
         exclusions: '*.git* README.md resources.tar.gz'
     - uses: ncipollo/release-action@v1
       with:
-        artifacts: "resources.tar.gz"
\ No newline at end of file
+        artifacts: "resources.tar.gz"
+        prerelease: true
+
+  build-deb:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get upload url
+        id: release_upload_url
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          result-encoding: string
+          script: |
+            let tag = context.payload.ref.replace(/.*\//, '');
+            let release =  await github.request("GET /repos/:owner/:repo/releases/tags/:tag", {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag: tag
+            });
+            console.log(release.data.upload_url);
+            return release.data.upload_url
+
+      - name: Prepare .deb directories
+        run: |
+          chmod +x debian/prepare-deb-build.sh
+          ./debian/prepare-deb-build.sh
+      - uses: jiro4989/build-deb-action@v3
+        id: deb
+        with:
+          package: ya-installer-resources
+          package_root: .debpkg
+          maintainer: "Golem Factory <contact@golem.network>"
+          version: ${{ github.ref }}
+          arch: 'amd64'
+          depends: 'golem-provider'
```

AFAIK `ya-provider pre-install` appeared in `0.13`
```suggestion
          depends: 'golem-provider (>= 0.13.0)'
```